android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationListenerService.java

```diff
@@ -93,6 +98,14 @@ private void sendNotification(ReactApplicationContext context, Bundle bundle) {
         if (bundle.getString("contentAvailable", "false").equalsIgnoreCase("true")) {
             jsDelivery.notifyRemoteFetch(bundle);
         }
+
+        Log.e(LOG_TAG, "RNPushNotificationListenerService.sendNotification: " + bundle);
+
+        if (!isRunning) {
```

Comment:
Ahh okay I dug a bit more now that I'm back at a computer, your code is the same as it was before my change.

isRunning and isApplicationRunning both check if the app is *foregrounded*, not merely that it is running. For the sake of better clarity of the old code, can you s/isApplicationRunning/isApplicationInForeground/ and s/isRunning/isForeground/?

Otherwise this change is a valid fix and I am fine with merging (though lack the power) to fix the issues people are seeing.